scss/_nav.scss

```diff
@@ -124,24 +145,30 @@
 //
 
 .nav-pills {
+  // scss-docs-start nav-pills-css-vars
+  --#{$prefix}nav-pills-border-radius: #{$nav-pills-border-radius};
+  --#{$prefix}nav-pills-link-active-color: #{$nav-pills-link-active-color};
+  --#{$prefix}nav-pills-link-active-bg: #{$nav-pills-link-active-bg};
+  // scss-docs-end nav-pills-css-vars
+
   .nav-link {
-    padding-right: $nav-tabs-link-padding-x;
-    padding-left: $nav-tabs-link-padding-x;
+    padding-right: var(--#{$prefix}nav-tabs-link-padding-x); // Boosted mod
+    padding-left: var(--#{$prefix}nav-tabs-link-padding-x); // Boosted mod
```

Comment:
These variables are not defined in the scope as we don't use nav-tabs here

Reply:
Fixed via 7cbfd895200103636bc5febc79c0724877cba440